app/views/shared/_navbar.html.erb

```diff
@@ -1,39 +1,32 @@
-<%# this is a boilerplate %>
-<div class="navbar navbar-expand-sm navbar-light navbar-lewagon">
+<div class="navbar fixed-bottom navbar-expand-sm navbar-light navbar-lewagon py-3">
+  <%# 4 icons + one collapse button in the navbar %>
   <div class="container-fluid">
-    <%= link_to "#", class: "navbar-brand" do %>
-      <%= image_tag "https://raw.githubusercontent.com/lewagon/fullstack-images/master/uikit/logo.png" %>
-    <% end %>
+    <a href="/"><i class="fas fa-home"></i></a>
+    <a href="/items"><i class="fas fa-search"></i></a>
+    <a href="/items/new"><i class="far fa-plus-square"></i></a>
+    <a href="/chatrooms"><i class="far fa-comment"></i></a>
 
+    <%# collapse button shows the avatar image (for the demo user only) or a normal icon (for everyone else) %>
     <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
-      <span class="navbar-toggler-icon"></span>
+      <% if current_user == User.first %>
+        <%= image_tag "demo_user.png", class: "avatar dropdown-toggle text-white", id: "navbarDropdown", data: { bs_toggle: "dropdown" }, 'aria-haspopup': true, 'aria-expanded': false %>
```

Comment:
Having an image next to a bunch of icons is weird, I would put the profile icon back

Reply:
ok, removed the profile picture as requested